### ISSUE TITLE
Upgrade from C108 -> C109 on macOS x64 via 1.48.x - Nightly (Checking C109)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#27354]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: `rs-ABPFilterParserData.dat` & `regional_catalog.json` (AdBlock)
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
~~- [ ] `jicbkmdloagakknpihibphagfckhjdih`: `speedreader-updater.dat`~~
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Rewards
    - [x] Wallet balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Wallet panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by LaurenWags** [comment]
Tests above verified with build from https://bravesoftware.slack.com/archives/CU5UXM3EC/p1671095392170929?thread_ts=1670971783.167499&cid=CU5UXM3EC

```
Brave | 1.48.56 Chromium: 109.0.5414.46 (Official Build) nightly (x86_64)
-- | --
Revision | 6e36b77363ef3febbe792af680fa1367993ddcf0-refs/branch-heads/5414@{#709}
OS | macOS Version 12.6.2 (Build 21G320)
```

**Comment by kjozwiak** [comment]
@bbondy mentioned that were was some signing issues, hence why we got the `macOS` builds a bit later as per @mkarolin. Quickly checked the following and ensured there were no notarization issue(s) using the following build(s):

```
Brave | 1.48.56 Chromium: 109.0.5414.46 (Official Build) nightly (x86_64) 
--- | ---
Revision | 6e36b77363ef3febbe792af680fa1367993ddcf0-refs/branch-heads/5414@{#709}
OS | macOS Version 13.0.1 (Build 22A400)
```

### Using `codesign -vvvv`

```
kjozwiak@Kamils-MBP-2 ~ % codesign -vvvv /Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly
--prepared:/Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/Current/.
--validated:/Applications/Brave Browser Nightly.app/Contents/Frameworks/Brave Browser Nightly Framework.framework/Versions/Current/.
/Applications/Brave Browser Nightly.app/Contents/MacOS/Brave Browser Nightly: valid on disk
/Applications/Brave Browser Nightly.app/Contents/MacOS/Brave Browser Nightly: satisfies its Designated Requirement
```

### Using `spctl -a -vv -t install`

```
kjozwiak@Kamils-MBP-2 ~ % spctl -a -vv -t install /Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly
/Applications/Brave Browser Nightly.app/Contents/MacOS/Brave Browser Nightly: accepted
source=Notarized Developer ID
origin=Developer ID Application: Brave Software, Inc. (KL8N8XSYF4)
```
